pkg/cluster/install.go

```diff
@@ -112,6 +112,10 @@ func (m *manager) Update(ctx context.Context) error {
 		steps.Action(m.clusterSPObjectID),
 		// credentials rotation flow steps
 		steps.Action(m.createOrUpdateClusterServicePrincipalRBAC),
+		steps.Action(m.startVMs),
```

Comment:
We still have `startVMs` step in `AdminUpdate`, but as far as I'm aware that is the current policy (decision by BU) is not to start VMs which were powered off.

Just something to check: Are we ok to have it in `Update`?

Reply:
I think there is a difference between SRE turning them on, and cx running the `az aro update` command. Jerome indicated in a slack thread he was ok with update performing the startVMs check.